Gems/Multiplayer/Code/Source/MultiplayerSystemComponent.cpp

```diff
@@ -972,7 +973,8 @@ namespace Multiplayer
 
         // Signal to session management when there are no remaining players in a dedicated server for potential cleanup
         // We avoid this for client server as the host itself is a user and non-transient dedicated servers
-        if (sv_isTransient && m_agentType == MultiplayerAgentType::DedicatedServer && connection->GetConnectionRole() == ConnectionRole::Acceptor)
+        // Replaced sv_isTransient with sv_terminateOnPlayerExit
```

Comment:
I think this comment can be removed given sv_isTransient is already marked DEPRECATED. That way the comments remain descriptive of the logic and not of naming history.

Reply:
@AMZN-puvvadar Just fixed this and made a commit